01-Introducing-Molecular-Dynamics/01b-Rigid-Body-Molecular-Dynamics-Simulations.ipynb

```diff
@@ -0,0 +1,362 @@
+{
```

Comment:
<p>You inconsistently capitalize eigenthings. I'm not sure which you prefer though but probably just pick one?</p><p> </p><p>"The Eigenvalues are the diagonalized principle moments of inertia of the body, while transpose of the eigenvectors is the rotation matrix to rotate the original particles into the frame where the inertia tensor from the frame of reference of the simulation is diagonalized." </p><p> </p><p>and</p><p> </p><p>"Rotating a rigid body by the rotation matrix by taking the dot product between the rotation matrix and the original position vectors of the particles. "</p><p> </p><p>are difficult to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>